pkgs/development/python-modules/bangla/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "bangla";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-F8j9UBMhZgB31atqebdGu6cfnkk573isDZp1171xXag=";
+  };
```

Comment:
The git repo has tests, bug no tags. Maybe we can poke the upstream to create tags?

Reply:
https://github.com/arsho/bangla/issues/5